### PR TITLE
adding capacity-mode-evaluator

### PR DESCRIPTION
**Pull request by Fedayizada**
*Issue #, if available:*

*Description of changes:*
adding Table Capacity Mode Evaluator to this repo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: capacity-mode-evaluator/README.md
Comment (switch180) [inline capacity-mode-evaluator/README.md:44]: Is this a minimum version, or the only version, or is there a range? Python 3.8 EOL is next year.

File: capacity-mode-evaluator/requirements.txt
Comment (switch180) [inline capacity-mode-evaluator/requirements.txt:2]: old

File: capacity-mode-evaluator/src/cost_estimates.py
Comment (switch180) [inline capacity-mode-evaluator/src/cost_estimates.py:7]: Costs aren't changed from region to region? We [hard code the costs](https://github.com/awslabs/amazon-dynamodb-tools/blob/main/reco/src/ddb_rc_reco/config.py#L9) in other parts of this package. If you're going to quote savings you need to be right. Perhaps we can move the cost matrix from reco up to a top level folder so that your code can use it? Or you can make a package that both tools can use to pull in cost data?

**Comment by switch180** [comment]
```
 % pip-audit -r capacity-mode-evaluator/requirements.txt 
Found 2 known vulnerabilities in 1 package
Name  Version ID                  Fix Versions
----- ------- ------------------- ------------
numpy 1.20.1  GHSA-6p56-wp2h-9hxr 1.21
numpy 1.20.1  GHSA-fpfv-jqm9-f5jm 1.22

```

File: capacity-mode-evaluator/README.md
Comment (Fedayizada) [reply capacity-mode-evaluator/README.md:44]: it's compatible with python versions 3.8 - 3.10.

File: capacity-mode-evaluator/requirements.txt
Comment (Fedayizada) [reply capacity-mode-evaluator/requirements.txt]: updated 
numpy==1.22.4
pandas==1.5.1

File: capacity-mode-evaluator/src/cost_estimates.py
Comment (Fedayizada) [reply capacity-mode-evaluator/src/cost_estimates.py]: added pricing function to retrieve pricing info based on user session region

File: capacity-mode-evaluator/src/metrics_estimates.py
Comment (tebanieo) [inline capacity-mode-evaluator/src/metrics_estimates.py:13]: `return j if i>j else i`

File: capacity-mode-evaluator/src/metrics_estimates.py
Comment (tebanieo) [inline capacity-mode-evaluator/src/metrics_estimates.py:6]: It is just a very minor change, but I would love to see this as one liner.

`return i if i>j else j`

File: capacity-mode-evaluator/src/metrics_estimates.py
Comment (tebanieo) [inline capacity-mode-evaluator/src/metrics_estimates.py:19]: These two methods do the same.

**Review by tebanieo — CHANGES_REQUESTED** [review]
One general suggestion, sometimes you use `camelCase` for variables and most of the time you use `snake_case`, I will prefer to see them all `snake_case` but it is just a cosmetic recommendation for easy reading. 

Check the other comments for more details.

File: capacity-mode-evaluator/README.md
Comment (tebanieo) [inline capacity-mode-evaluator/README.md:136]: Suggestion:

**_Note:_**

The analysis provided in this script compares your table consumption and simulates cost using different parameters. This tool does not have access to your contextual information, business requirements or organization best practices. When changing your capacity mode from on-demand to provisioned based on the results, remember there were some assumptions made: 
 - The analysis window is 14 days.
 - Auto-scaling responds instantaneously. (In reality, Auto scaling service might take 4 mins to provision new table capacity depending on your increase conditions).

File: capacity-mode-evaluator/capacity_reco.py
Comment (Fedayizada) [inline capacity-mode-evaluator/capacity_reco.py:88]: @tebanieo

**Review by tebanieo — APPROVED** [review]

**Comment by switch180** [comment]
Are we good to squash and merge this? I'm okay with it.

**Comment by tebanieo** [comment]
It is usually the opener responsibility to squash and merge after you get all the approvals @Fedayizada. @switch180 I thought this branch was already merged.

**Pull request merged**
